src/test/java/parser/GfaParserTest.java

```diff
@@ -16,7 +16,7 @@
 
     GfaParser parser;
     SequenceGraph graph;
-
+/*
```

Comment:
Because we don't have enough time to test now this is fine. Lets put it in the sprint backlog.